server/models/api.go

```diff
@@ -138,3 +138,8 @@ func (api *Api) Update(message *rpc.Api, mask *fieldmaskpb.FieldMask) error {
 	api.UpdateTime = time.Now()
 	return nil
 }
+
+// LabelsMap returns a map representation of stored labels.
+func (api *Api) LabelsMap() (map[string]string, error) {
+	return mapForBytes(api.Labels)
+}
```

Comment:
No change necessary, just thinking out loud. As more fields appear with serialized storage formats it seems increasingly appealing to have a data access package that insulates the RPC handlers from handling all the storage logic, making internal types usable immediately after retrieval.

Reply:
To some extent the "models" package helps with this, but further improvement sgtm.